src/lib/components/icon/QIcon.svelte

```diff
@@ -16,16 +15,13 @@
     userStyles: QIconProps["userStyles"] = undefined;
   export { userClasses as class, userStyles as style };
 
-  $: sizeStyle = useSize(size) === null ? (isNumber(size) ? `${size}px` : size) : undefined;
+  $: sizeStyle = isNumber(size) ? `${size}px` : undefined;
```

Comment:
I wanted users to be able to pass custom CSS sizes here (like 3em or 70% for example). Please, change back to the original code, or you have a better idea?